src/summary/history/patient-notes/patient-notes-card-level-2.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { match } from "react-router";
+import NotesCard from "./notes-card.component";
+import Styles from "./patient-notes-style.css";
+import { getEncounters } from "./encounter.resource";
+
+export default function PatientNotes(props: PatientNotesCardProps) {
```

Comment:
Components should be named the same as the their filename. This makes it easy to find where components are defined:

```suggestion
export default function PatientNotesCardLevel2(props: PatientNotesCardProps) {
```